go.mod

```
@@ -3,7 +3,7 @@ module github.com/kopia/kopia
 go 1.22.7
```

```suggestion
go 1.22
```

To close https://github.com/kopia/kopia/issues/4292 this pr needs modifying or follow up.

---

Please open a separate PR.